composer/algorithms/copypaste/copypaste.py

```diff
@@ -0,0 +1,403 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Core CopyPaste classes and functions."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any, Callable, Optional, Tuple, Union
+
+from numpy.random import default_rng
+import numpy as np
+import random
+import torch
+import torchvision.transforms as T
```

Comment:
```suggestion
import torchvision.transforms as transforms
```
I think the convention is to just use `transforms` as the module name.

Reply:
it's common to use either. Here's are transforms imported as T in the official docs. https://pytorch.org/vision/stable/auto_examples/plot_scripted_tensor_transforms.html#sphx-glr-auto-examples-plot-scripted-tensor-transforms-py